### DISCUSSION TITLE
RC522 - What format should be data stored in NFC tag

### DISCUSSION THREAD

**Discussion by jim-022** [arendst/Tasmota#20774] [Q&A]
Hello,

I have a setup with nodemcu and RC522, all works fine. I am able to detect tags and Read UIS from tasmota. 
![image](https://github.com/arendst/Tasmota/assets/6962522/bccedf98-0042-48d0-909f-cc0b898e3ec4)

I have tried using android phone with NFC tools pro to write plain text records on the tags/

My queston is how should the data be stored in the tag so tasmota can display it in the console?

**Comment by Noschvie** [comment]
@MartinAngelov Have you found a solution ?

**Reply by jim-022** [reply]
@Noschvie , not yet. I will be checking it again this week.
